<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FWLite</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="console">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Local Rules</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>28</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="LocalRulesLayout"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Add Rule</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LocalRuleEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Expires</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ExpireEdit">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Minutes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddLocalRuleButton">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="uriEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="isgfwedTestButton">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Redirector Rules</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>28</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="RedirectorRulesLayout"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Add Rule</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="RuleEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Redirect to</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DestEdit">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddRedirectorRuleButton">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>ProxyList</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="60,40">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QTableView" name="proxyListView">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="proxyActivateButton">
              <property name="text">
               <string>Activate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="proxyDisableButton">
              <property name="text">
               <string>Disable</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="proxyRemoveButton">
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,0">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="title">
             <string>Add Proxy</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QFormLayout" name="formLayout_4">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="1" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Name</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="nameEdit">
                 <property name="inputMethodHints">
                  <set>Qt::ImhNone</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Hostname</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="hostnameEdit"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Port</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="portEdit">
                 <property name="inputMethodHints">
                  <set>Qt::ImhDigitsOnly</set>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QComboBox" name="encryptionBox"/>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>PSK</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QLineEdit" name="pskEdit">
                 <property name="echoMode">
                  <enum>QLineEdit::PasswordEchoOnEdit</enum>
                 </property>
                </widget>
               </item>
               <item row="10" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>Priority</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="10" column="1">
                <widget class="QLineEdit" name="priorityEdit">
                 <property name="inputMethodHints">
                  <set>Qt::ImhDigitsOnly|Qt::ImhNoPredictiveText</set>
                 </property>
                 <property name="text">
                  <string>99</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Encryption</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Protocol</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="protocolBox"/>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="usernameEdit"/>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="passwordEdit">
                 <property name="echoMode">
                  <enum>QLineEdit::PasswordEchoOnEdit</enum>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Plugin</string>
                 </property>
                </widget>
               </item>
               <item row="9" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Plugin-opt</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QComboBox" name="pluginBox"/>
               </item>
               <item row="9" column="1">
                <widget class="QLineEdit" name="plugin_optEdit"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Username</string>
                 </property>
                </widget>
               </item>
               <item row="11" column="0">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Via</string>
                 </property>
                </widget>
               </item>
               <item row="11" column="1">
                <widget class="QLineEdit" name="viaEdit"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item>
             <widget class="QPushButton" name="proxyAddButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exclusiveProxyAddButton">
              <property name="text">
               <string>Add and disable other Proxies</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Port Forward</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="4,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Port Forward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="PFView">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="PFRemoveButton">
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14" stretch="1,0">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="title">
             <string>Add Port Forward</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Target</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>Local Port</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Proxy</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="PFProxyBox">
                 <property name="editable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="PFPortEdit">
                 <property name="inputMethodHints">
                  <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="PFTargetEdit">
                 <property name="inputMethodHints">
                  <set>Qt::ImhPreferLatin|Qt::ImhUrlCharactersOnly</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PFAddButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="1" column="0">
             <widget class="QCheckBox" name="gfwlistToggle">
              <property name="text">
               <string>GFWList</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="adblockToggle">
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>ADBlock</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="sys_proxy_toggle">
              <property name="text">
               <string>auto_system_proxy</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="startup_toggle">
              <property name="text">
               <string>Run at Startup</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="editConfButton">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Edit userconf.ini</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editLocalButton">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Edit local.txt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
